db/migrate/20210419233225_create_microsoft_auth.rb

```diff
@@ -0,0 +1,6 @@
+class CreateMicrosoftAuth < ActiveRecord::Migration[5.2]
+  def change
+   add_column :teachers, :microsoft_token, :string
+   add_column :teachers, :microsoft_refresh_token, :string
```

Comment:
Re previous comment, is it necessary to distinguish between microsoft_token and google_token?